Primo.js

```diff
@@ -759,6 +777,34 @@ var testCases = [
 				"seeAlso": []
 			}
 		]
+	},
+	{
+		"type": "web",
+		"url": "http://digitale.beic.it/primo_library/libweb/action/display.do?doc=39bei_digitool2018516",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Grida per i Milanesi che avevano seguito Ludovico il Moro",
+				"creators": [
+					{
+						"lastName": "Milano",
+						"creatorType": "author",
+						"fieldMode": 1
+					}
+				],
+				"date": "1500",
+				"language": "ita",
+				"libraryCatalog": "digitale.beic.it",
+				"place": "Milano",
+				"publisher": "Ambrogio : da Caponago",
```

Comment:
sorry, but the regex you had (and anything I could think to use for the same effect) simply caught too much and is too risky to implement in such a widely used translator.